### DISCUSSION TITLE
Testbed GUI buttons not responding to mouse click

### DISCUSSION THREAD

**Discussion by miquelrosell99** [NVlabs/instant-ngp#509] [Q&A]
Hi,
I want to stop the training process, but the buttons don't seem to respond to mouse clicks... Any way to fix this?
I can't stop training, or export, or anything.

Best regards,

**Comment by pwais** [comment]
it could be your GPU is slow and/or the render is slow.  if you click and hold, eventually the mouse event will happen; there might be a 1-2 sec delay sometimes
